oldfal/check_usernames.py

```diff
@@ -0,0 +1,36 @@
+#!/usr/bin/python3.7
```

Comment:
This is new, but I think the old version is in a previous repo. Should we put that in here to preserve the true old version?